src/Pages/OfficerDB/OfficerDB.js

```diff
@@ -1,75 +1,111 @@
 import React from 'react'
-import './officer-db.css'
+import './OfficerDB.css'
 import 'bootstrap/dist/css/bootstrap.min.css'
-import { Table } from 'reactstrap'
+import { Table, Input } from 'reactstrap'
+import TableRow from './InfoRow'
```

Comment:
Either rename the file as TableRow or do:
```suggestion
import InfoRow from './InfoRow'
```